modules/plugin/shapefile/src/test/java/org/geotools/data/shapefile/TestCaseSupport.java

```diff
@@ -16,7 +16,7 @@
  */
 package org.geotools.data.shapefile;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
No wildcard imports please (there is a QA rule to prevent it, but was not running for tests, so far... the PMD support for tests just landed)